cluster/gce/gci/configure-helper.sh

```diff
@@ -286,31 +286,30 @@ function create-master-auth {
   local -r auth_dir="/etc/srv/kubernetes"
   local -r basic_auth_csv="${auth_dir}/basic_auth.csv"
   if [[ -n "${KUBE_PASSWORD:-}" && -n "${KUBE_USER:-}" ]]; then
+    # Remove basic_auth_csv because we will rewrite it.
```

Comment:
Can you put this comment one line below and be a little more specific (because this bash is so gross).

Maybe:
When METADATA_CLOBBERS_CONFIG==true, just remove the basic auth file, because the append_or_replace calls below will regenerate the file.